.github/workflows/sims.yml

```diff
@@ -0,0 +1,183 @@
+name: Sims
+# Sims workflow runs multiple types of simulations (nondeterminism, import-export, after-import, multi-seed-short)
+# This workflow will run on main and release branches, if a .go, .mod or .sum file have been changed
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+    branches:
+      - main
+      - release/**
+    tags:
+      - "*"
+  pull_request:
+    types: auto_merge_enabled
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: >-
+      (github.event_name == 'issue_comment' && github.event.comment.body == '/runsim') ||
+      github.event_name == 'push' || github.event_name == 'pull_request'
+    steps:
+      - name: Check if organization member
+        uses: JamesSingleton/is-organization-member@main
+        id: checkMember
+        with:
+          username: ${{ github.actor }}
+          organization: crypto-org-chain
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Comment PR for authentication failure
+        uses: allthatjazzleo/actions-pull-request-add-comment@master
```

Comment:
it'd be better to use the org repo instead of a personal repo for this action
https://github.com/crypto-org-chain/actions-pull-request-add-comment/